pre_processing/set_ecmwf.F90

```diff
@@ -93,45 +93,36 @@ subroutine set_ecmwf(granule, opts, ecmwf_flag, imager_geolocation, &
    character     :: cera_year*4, cera_month*2, cera_day*2, cera_hour*2
    character     :: cera_year2*4, cera_month2*2, cera_day2*2, cera_hour2*2
 
-   real(dreal)   :: time_fac
```

Comment:
I can see why you made this a driver file argument but, like ecmwf_nlevels, this is going to be a thing that I always forget needs to be changed. BUT this one won't crash the code if I get it wrong. (a) It needs documenting with the other command line arguments. (b) Would be worth thinking about if there's any way to deduce it for assume_full_path.